build/build-i18n.js

```diff
@@ -0,0 +1,69 @@
+const fs = require('fs');
+
+const outFilePath = './i18n';
+const umdWrapperHead = `
+(function(root, factory) {
+    if (typeof define === 'function' && define.amd) {
+        // AMD. Register as an anonymous module.
+        define(['exports'], factory);
+    } else if (
+        typeof exports === 'object' &&
+        typeof exports.nodeName !== 'string'
+    ) {
+        // CommonJS
+        factory(exports);
+    } else {
+        // Browser globals
+        factory({});
+    }
+})(this, function(exports) {
+var lang =`;
```

Comment:
Seems there is no export here.

Also obj export is not convenient in the case which loads script with HTML tag. I think we can keep it simple and only use CommonJS exports for the `-obj` files.

Reply:
the exports code at the L54 like 
```